src/project-impact-graph.yaml

```diff
@@ -2,11 +2,11 @@ globalExcludedGlobs:
     - OWNERS
```

Comment:
Is these file names still working?  The implementation has been changed to match with `startsWith`